foundations/html_css/html-foundations/project-recipes.md

```diff
@@ -99,4 +92,3 @@ There are a couple of ways to go about doing this, but the simplest is this:
 -   Click on **Pages** on the left side bar.
 -   Change the **Branch** from _none_ to _main branch_ and click **Save**.
 -   It may take a few minutes (The GitHub website says up to 10, but we've seen it take up to an hour. Do not add a "theme" to your project, or you may have git conflicts, instead, be patient.) but your project should be accessible over the web from `your-github-username.github.io/your-github-repo-name` (obviously substituting your own details in the link).
-
```

Comment:
This empty line should be kept

Reply:
Are you referring to the very last line?